static/app/views/performance/browser/resources/resourceSidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import omit from 'lodash/omit';
+import {PlatformIcon} from 'platformicons';
+
+import FileSize from 'sentry/components/fileSize';
+import useRouter from 'sentry/utils/useRouter';
+import {BrowserStarfishFields} from 'sentry/views/performance/browser/resources/utils/useResourceFilters';
+import DetailPanel from 'sentry/views/starfish/components/detailPanel';
+import {Block, BlockContainer} from 'sentry/views/starfish/views/spanSummaryPage/block';
+
+type Props = {
+  description?: string;
+  groupId?: string;
```

Comment:
I think we only need group here - feel free to do in the follow up PR

Reply:
Yup good call, we can get all the other info from group. i'll do it in a follow up!